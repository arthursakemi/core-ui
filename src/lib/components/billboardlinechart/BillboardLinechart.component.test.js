import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/billboardlinechart/__snapshots__"
});
