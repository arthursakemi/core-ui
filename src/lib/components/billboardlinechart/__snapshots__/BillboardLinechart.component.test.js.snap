// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots billboard-linechart Default 1`] = `
<div>
  <h3>
    billboard linechart demo (supervisor)
  </h3>
  <div
    className="sc-bdVaJa gWtJaj sc-billboardlinechart"
    id="supdragZoom"
  />
  <h3>
    billboard linechart performance test
  </h3>
  <div
    className="sc-bdVaJa gWtJaj sc-billboardlinechart"
    id="dragZoom"
  />
</div>
`;
